tests/bugs/shard/issue-1425.t

```diff
@@ -21,7 +21,13 @@ TEST $CLI volume profile $V0 start
 
 TEST $GFS --volfile-id=$V0 --volfile-server=$H0 $M0
 
+echo `$CLI volume profile $V0 info incremental | grep -w LOOKUP | awk '{print $8}'` > /dev/zero
+
 TEST fallocate -l 20M $M0/foo
+
+# There should be 1+4 shards and we expect 4 lookups less than on the build without this patch
```

Comment:
Not relevant to this patch. 

The reason for this change is that the test was failing at line 34 because of inconsistent lookup numbers. I am moving that check once the fallocate is done to minimize the window and I think that's what test 1425 also expects